### PR TITLE
AST-156: Add Common issues section

### PR DESCRIPTION
**Pull request by SamirBoulil**
Adds a "common issues", and "Pre-requisite" sections in the readme.

### CODE REVIEW

File: mapping.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (pierallard) [inline mapping.json:1]: Pourquoi ? Il y a déjà mapping.json.dist.

File: src/Command/CreateFamilyCommand.php [hunk 2]
```diff
@@ -120,6 +120,16 @@ public function execute(InputInterface $input, OutputInterface $output)
             $categoryOptions = array_filter(explode(',', $categoryOptions), function (string $categoryOption) {
                 return !empty($categoryOption);
             });
+            $numberOfOptions = \count($categoryOptions);
+            if ($numberOfOptions > 100) {
+                $this->io->warning(
```
Comment (pierallard) [inline src/Command/CreateFamilyCommand.php:125]: if it fail, it's an error.

**Review by pierallard — APPROVED** [review]

**Review by juliensnz — APPROVED** [review]

File: mapping.json (hunk 1 above)
Comment (SamirBoulil) [reply mapping.json:1]: erreur :/ je voulais pas jsui désolé

File: src/Command/CreateFamilyCommand.php (hunk 2 above)
Comment (SamirBoulil) [reply src/Command/CreateFamilyCommand.php:125]: Yes, when the user sets the option to convert the categories to multiselect but at the same time has more than 100, it fails by explaning him why.

It does not really change the behavior does it ?

**Pull request merged**